R/guides-axis.r

```diff
@@ -47,24 +47,24 @@ guide_axis <- function(at, labels, position = "right", theme) {
     left =  element_render(theme, label_render, labels, y = label_y, expand_x = TRUE))
 
   line <- switch(position,
-    top =    element_render(theme, "axis.line.x", c(0, 1), c(0, 0), id.lengths = 2),
+    top =    element_render(theme, "axis.line.x.top", c(0, 1), c(0, 0), id.lengths = 2),
```

Comment:
yeah, I thought about that too as I revised it now... My sentiment is that the top and right are special cases that deserves a qualifier, while it is expected that `axis.*.x` defaults to bottom etc.